src/aks-preview/azext_aks_preview/_validators.py

```diff
@@ -585,3 +585,11 @@ def validate_azure_keyvault_kms_key_id(namespace):
         segments = key_id[len(https_prefix):].split("/")
         if len(segments) != 4 or segments[1] != "keys":
             raise InvalidArgumentValueError(err_msg)
+
+
+def validate_enable_custom_ca_trust(namespace):
+    """Validates Custom CA Trust can only be used on Linux."""
+    if namespace.enable_custom_ca_trust is not None and namespace.enable_custom_ca_trust:
```

Comment:
No need to check if it's `None`, since we only care the option is `True`, which means this equal to?
```suggestion
    if namespace.enable_custom_ca_trust:
```